names/names.go

```diff
@@ -146,57 +146,7 @@ const (
 )
 
 const (
-	// AWS Standard partition's regions.
 	GlobalRegionID = "aws-global" // AWS Standard global region.
```

Comment:
Not blocking, but consider replacing uses of `names. GlobalRegionID` with `endpoints.AwsGlobalRegionID`